app/script/user/UserService.js

```diff
@@ -0,0 +1,290 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+'use strict';
+
+window.z = window.z || {};
+window.z.user = z.user || {};
+
+z.user.UserService = class UserService {
+  static get URL() {
+    return {
+      CONNECTIONS: '/connections',
+      PROPERTIES: '/properties',
+      SELF: '/self',
+      USERS: '/users',
+    };
+  }
+
+  /**
+   * Construct a new User Service.
+   * @class z.user.UserService
+   * @param {z.service.BackendClient} client - Client for the API calls
+   */
+  constructor(client) {
+    this.client = client;
+    this.logger = new z.util.Logger('z.search.SearchService', z.config.LOGGER.OPTIONS);
+  }
+
+  /**
+   * Create a connection request to another user.
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/users/createConnection
+   * @param {string} user_id - User ID of the user to request a connection with
+   * @param {string} name - Name of the conversation being initiated (1 - 256 characters)
+   * @returns {Promise} Promise that resolves when the connection request was created
+   */
+  create_connection(user_id, name) {
+    return this.client.send_json({
+      data: {
+        message: ' ',
+        name,
```

Comment:
But sometimes we use the short form and sometimes we don't... I also prefer the explicit one so what does the team want to use? @lipis @herrmannplatz @ffflorian ?